tools/travis-test.sh

```diff
@@ -25,7 +25,7 @@ if [ -n "$PYTHON_OPTS" ]; then
 fi
 
 # make some warnings fatal, mostly to match windows compilers
-werrors="-Werror=vla -Werror=nonnull -Werror=pointer-arith"
+werrors="-Werror=vla -Werror=nonnull -Werror=pointer-arith -Werror=implicit-function-declaration"
```

Comment:
Best to break the long quote with `\`, no spaces after the backslash. There are other tricks, but that should work.

Reply:
There is also the plus equals form
```
a = "hello"
a = "$a world"
```

Reply:
ok, I adopted the second approach